test/unit/jest.config.json

```diff
@@ -1,6 +1,7 @@
 {
 	"rootDir": "../../",
 	"moduleNameMapper": {
+		"@wordpress\\/(browserslist-config)$": "@wordpress/$1",
```

Comment:
I have PR coming up soon which will redefine `moduleNameMaper` completely to make it bullet-proof, so let's fly with this for now.